lib/utils.js

```diff
@@ -387,3 +398,38 @@ function prettifyTime ({ log, timestampKey = TIMESTAMP_KEY, translateFormat = un
 
   return `[${time}]`
 }
+
+/**
+ * Prettifies an error string into a multi-line format.
+ * @param {object} input
+ * @param {string} input.keyName The key assigned to this error in the log object
+ * @param {string} input.lines The STRINGIFIED error. If the error field has a
+ *  custom prettifier, that should be pre-applied as well
+ * @param {string} input.ident The indentation sequence to use
+ * @param {string} input.eol The EOL sequence to use
+ */
+function prettifyError ({ keyName, lines, eol, ident }) {
```

Comment:
This function should be added to `module.exports.internals` and unit tests written to specifically cover it.